src/components/Categories.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Categories() {
+  return (
+    <div>
+      <h1>Categories Page</h1>
+    </div>
+  );
```

Comment:
- [ ] According to the requirements the Categories component should contain a button that checks for the status.

Reply:
Hey @mikethreels, Thanks for your review I will implement all suggested changes, but I didn't understand this part, what kind of button should it check for which type of status? 
Please make it more clear.